### PR TITLE
Add AlmaLinux OS Documentation

### PR DESCRIPTION
**Pull request by ezamriy**
Add documentation for the new, official AlmaLinux OS images.

### CODE REVIEW

**Review by yosifkit — APPROVED** [review]

File: almalinux/content.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# AlmaLinux OS
+
+[AlmaLinux OS](https://almalinux.org/) is an Open Source and forever-free enterprise Linux distribution, governed and driven by the community, focused on long-term stability and a robust production-grade platform. AlmaLinux OS is a 1:1 binary compatible with RHEL® 8 and it was founded by the team behind the well-established [CloudLinux OS](https://www.cloudlinux.com/all-products/product-overview/cloudlinuxos).
+
+%%LOGO%%
+
+# About this image
+
+The `almalinux:latest` tag will always point to the latest stable release. Major releases are also tagged with their version (e.g. `almalinux:8`). All images for supported releases will be updated monthly or as needed for security fixes.
```
Comment (tianon) [inline almalinux/content.md:9]:
```suggestion
The `%%IMAGE%%:latest` tag will always point to the latest stable release. Major releases are also tagged with their version (e.g. `%%IMAGE%%:8`). All images for supported releases will be updated monthly or as needed for security fixes.
```

File: almalinux/content.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# AlmaLinux OS
+
+[AlmaLinux OS](https://almalinux.org/) is an Open Source and forever-free enterprise Linux distribution, governed and driven by the community, focused on long-term stability and a robust production-grade platform. AlmaLinux OS is a 1:1 binary compatible with RHEL® 8 and it was founded by the team behind the well-established [CloudLinux OS](https://www.cloudlinux.com/all-products/product-overview/cloudlinuxos).
```
Comment (tianon) [inline almalinux/content.md:3]:
```suggestion
[AlmaLinux OS](https://almalinux.org/) is an Open Source and forever-free enterprise Linux distribution, governed and driven by the community, focused on long-term stability and a robust production-grade platform. AlmaLinux OS is 1:1 binary compatible with RHEL® 8 and it was founded by the team behind the well-established [CloudLinux OS](https://www.cloudlinux.com/all-products/product-overview/cloudlinuxos).
```

File: almalinux/content.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+%%LOGO%%
+
+# About this image
+
+The `almalinux:latest` tag will always point to the latest stable release. Major releases are also tagged with their version (e.g. `almalinux:8`). All images for supported releases will be updated monthly or as needed for security fixes.
+
+## How It's Made
+
+The rootfs tarballs for this image are built using the [livemedia-creator tool](http://weldr.io/lorax/livemedia-creator.html). The build script and kikstart files are available in the [AlmaLinux/docker-images](https://github.com/AlmaLinux/docker-images) git repository.
```
Comment (tianon) [inline almalinux/content.md:13]:
```suggestion
The rootfs tarballs for this image are built using the [livemedia-creator tool](http://weldr.io/lorax/livemedia-creator.html). The build script and kickstart files are available in the [AlmaLinux/docker-images](https://github.com/AlmaLinux/docker-images) git repository.
```

**Review by tianon — APPROVED** [review]
A few minor suggestions, but no blockers :+1:

**Comment by ezamriy** [comment]
Thanks for your recommendations! I've submitted #1951 to fix it.

**Pull request merged**